pytest.ini

```diff
@@ -14,6 +14,7 @@ filterwarnings =
 # --last-failed to run only the tests which failed on the previous run
 # --show-capture no/stdout/stderr/both
 # -k "test name" will run only the tests that match the pattern in quotes, i.e. "test_arkane or test_rmg" would run those two tests
+# -n auto # will use all available cores, if you first pip install pytest-xdist, but it doesn't work well with julia (so add --no-julia)
```

Comment:
I also tried running in parallel and ended up with Julia issues. I think `--no-julia` won't work for us though - we would need to individually label each unit test with either a pytest marker or fixture to indicate that it requires the Julia runtime (see the [`pyjulia` docs](https://pyjulia.readthedocs.io/en/latest/pytest.html#fixture)).

Reply:
Thanks. I tweaked the comment and added more details in the commit message.
I'll leave it there for people to explore, as there are cases it might be useful, but is unfortunately not a good default.
Did I mention that I hate PyJulia?

Reply:
![image](https://github.com/ReactionMechanismGenerator/RMG-Py/assets/33505528/d7d2c131-fdbc-443c-b42a-3c304ebc1c86)